src/Apps/Feature/Components/FeatureHeader.tsx

```diff
@@ -1,67 +1,70 @@
 import React from "react"
 import styled from "styled-components"
-import {
-  Box,
-  Flex,
-  Join,
-  ResponsiveImage,
-  Sans,
-  Spacer,
-  color,
-} from "@artsy/palette"
+import { Flex, FlexProps, Join, Sans, Spacer, color } from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
 import { FeatureHeader_feature } from "__generated__/FeatureHeader_feature.graphql"
+import { NavBarHeight } from "Components/NavBar"
+import { HTML } from "Components/HTML"
 
 const Container = styled(Flex)`
-  height: 100%;
   width: 100%;
   flex: 1;
+  border-bottom: 1px solid ${color("black100")};
 `
 
 const Figure = styled(Flex)`
   flex-basis: 50%;
   flex-shrink: 0;
   flex-grow: 0;
   background-color: ${color("black10")};
+  overflow: hidden;
 `
 
 const Meta = styled(Flex)`
-  flex-shrink: 0;
+  flex-shrink: 1;
   flex-grow: 0;
   flex-direction: column;
   align-items: center;
   justify-content: center;
 `
 
-export type FeatureHeaderProps = {
+export interface FeatureHeaderProps extends FlexProps {
   feature: FeatureHeader_feature
 }
 
 export const FeatureHeader: React.FC<FeatureHeaderProps> = ({
   feature: { name, subheadline, image },
+  ...rest
 }) => {
   return (
-    <Container flexDirection={["column", "row"]}>
+    <Container
+      display={["block", "flex"]}
+      height={["auto", !!image ? `calc(95vh - ${NavBarHeight}px)` : "50vh"]}
+      {...rest}
+    >
       {image && (
-        <Figure>
-          <ResponsiveImage
-            src={image.cropped.url}
-            width="100%"
-            style={{ backgroundSize: "cover", height: "100%" }}
-          />
-        </Figure>
+        <Figure
+          style={{
+            backgroundImage: `url(${image.cropped.url})`,
+            backgroundSize: "cover",
+            backgroundPosition: "center center",
+          }}
+        />
       )}
 
-      <Meta p={6} flexBasis={image ? "50%" : "100%"}>
+      <Meta p={4} flexBasis={image ? "50%" : "100%"}>
         <Join separator={<Spacer my={1} />}>
           <Sans size="10" element="h1" textAlign="center">
             {name}
           </Sans>
 
           {subheadline && (
-            <Sans size="4" textAlign="center">
-              <Box dangerouslySetInnerHTML={{ __html: subheadline }} />
-            </Sans>
+            <HTML
+              fontFamily="sans"
+              size="4"
+              textAlign="center"
+              html={subheadline}
+            />
```

Comment:
Should this be in Palette?

Reply:
(Not a blocker of course)

Reply:
Probably — it sort of seems like this doesn't come up much (?) but it'd probably be nice to have a comprehensive set of styles for all the elements we support in Markdown (h1, h2, etc). My main purpose here was just to neutralize the inner margins.